src/coreclr/gcinfo/gcinfodumper.cpp

```diff
@@ -392,6 +396,9 @@ PORTABILITY_ASSERT("GcInfoDumper::ReportPointerRecord is not implemented on this
 #elif defined(TARGET_LOONGARCH64)
     assert(!"unimplemented on LOONGARCH yet");
                 pReg = (SIZE_T*)(pContext + rgRegisters[iReg].cbContextOffset);
+#elif defined(TARGET_RISCV64)
+    assert(!"unimplemented on RISCV64 yet");
+                pReg = (SIZE_T*)(pContext + rgRegisters[iReg].cbContextOffset);
```

Comment:
Note, this is unimplemented block (and will not be implemented in this PR for sure). We stay with previous logic - assert in debug build and compile default code in release build. I can't remove any of this lines since this will broke previous logic - will not assert in debug or will not compile code in `#else` block.

Reply:
Sorry. I cannot understand well. If the block is unimplemented, I don't understand why we need those default codes for release build? And `pReg = (SIZE_T*)(pContext + rgRegisters[iReg].cbContextOffset);` is necessary for only Release build?  Could you explain for me?

Reply:
We have 2 situation here: 1) without some of this code blocks you can't compile, since this blocks provide local variables definitions; 2) I can't guarantee that in release build, code will avoid "infinity loop" or something like that, in case we don't change some variables, that changed by initial logic;

This mean, in case someone will use diagnostics SOS command `gcinfo` (after this code will be synced with diagnostics repo) it should not hangs or sigsegv in release build.